packages/enzyme/src/Debug.js

```diff
@@ -32,7 +33,11 @@ function propString(prop) {
     case 'boolean':
       return `{${prop}}`;
     case 'object':
-      return '{{...}}';
+      if (isString(prop)) {
+        return `"${prop}"`;
+      } else {
```

Comment:
The else here isn’t necessary; the following return can be de-indented

Reply:
👍 updated